### PR TITLE
Fix 404 Error

### PR DESCRIPTION
**Pull request by meixinzhang**
Fix 404 Error and added an explanation for tf.Example

@zhitaoli

### CODE REVIEW

File: docs/guide/bulkinferrer.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ contains the original features and the prediction results.
 
 BulkInferrer consumes:
 
-*   A Trained model in [SavedModel](/guide/saved_model) format.
-*   Validation result from [Evaluator](evaluator) component.
-*   Unlabeled tf.Examples that contain features.
+*   A Trained model in [SavedModel](/guide/saved_model.md) format.
+*   Validation result from [Evaluator](evaluator.md) component.
+*   Unlabeled Unlabelled [tf.Examples](https://www.tensorflow.org/tutorials/load_data/tfrecord.md) that contain features.
```
Comment (numerology) [inline docs/guide/bulkinferrer.md:12]: Redundant 'Unlabeled'?

Comment (meixinzhang) [reply docs/guide/bulkinferrer.md]: Good catch! Now Fixed

File: docs/guide/bulkinferrer.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ contains the original features and the prediction results.
 
 BulkInferrer consumes:
 
-*   A Trained model in [SavedModel](/guide/saved_model) format.
-*   Validation result from [Evaluator](evaluator) component.
-*   Unlabeled tf.Examples that contain features.
+*   A Trained model in [SavedModel](/guide/saved_model.md) format.
```
Comment (numerology) [inline docs/guide/bulkinferrer.md:10]: Sorry did not catch this. `/guide/saved_model.md` seems to be non-existence.

@zhitaoli Do you happen to know where should this be pointed? Perhaps https://www.tensorflow.org/guide/saved_model ?

Comment (meixinzhang) [reply docs/guide/bulkinferrer.md:10]: Must've been changed, should I change all of them to https://www.tensorflow.org/guide as it might be more stable

**Pull request merged**